considerations.md

```diff
@@ -134,9 +131,15 @@ The production "component" is one or more lowercase letters.
 A "path" is then at least one component, possibly followed by zero or more slash-component pairs.
 The above can be converted into the following regular expression:
 
-```
+```text
 [a-z]+(?:/[a-z]+)*
 ```
```

Comment:
Ooh, I love explicit types on these -- should this one be `regex`/`regexp`? :eyes: (it's not much, but it's something)

```regex
[a-z]+(?:/[a-z]+)*
```

````suggestion
```regex
[a-z]+(?:/[a-z]+)*
```
````

Reply:
I was looking for that in the github syntax highlighting and couldn't find it. Apparently it uses https://highlightjs.org/static/demo/. Is regex/regexp supported in other markdown renderers?

Reply:
:shrug: worst case they render it as plain text, right?  I don't think I've seen one that renders something unknown with an error or in some unexpected way :sweat_smile:

(I think that https://github.com/github-linguist/linguist/blob/master/lib/linguist/languages.yml is the GitHub set of supported languages, which is where I found https://github.com/github-linguist/linguist/blob/e855ef2b6f90c34074061a2e17acbe853e61b483/lib/linguist/languages.yml#L5859-L5870)

Reply:
Looks good. Fixed!